spec/features/datepicker_spec.rb

```diff
@@ -3,6 +3,15 @@
 RSpec.feature "visitor selects a date" do
   include_examples "feature helper"
 
+  before :all do
+    next_tuesday = (Time.new + 1.week).beginning_of_week.at_noon + 1.day
```

Comment:
Does `beginning_of_week` always give us a Monday, or does it change with locale?